js/app.js

```diff
@@ -11,6 +11,26 @@ alert("You will answer with a yes or no");
 let points = 0;
 
 
+// let answer = prompt("Do I Like Sports");
+
+
+function sports (answer){
+
+if (answer.toLowerCase() === "yes") alert("Yes I like Sports!");
+else if (answer.toLowerCase() === "no") alert("Wrong Of Course I Like Sports!");
+// console.log(answer);
+}
+sports(answer);
+
+
+let answer2 = prompt("Do I Love Insects?");
+
+function insects(answer2){
+if (answer2.toLowerCase() === "no") alert("You are correct I hate bugs!");
+else if (answer2.toLowerCase() === "yes") alert("That is not correct I dont like bugs!");
+// console.log(answer2);
+}
+insects(answer2);
```

Comment:
Try to put all your functional code in one spot. Typically all your function calls will go at the bottom of a file.